README.md

```diff
@@ -89,7 +89,7 @@ To build a project with Maven artifacts, use `mvn clean install`
 
 To build a project with Gradle artifacts, use `gradle build publishToMavenLocal`
 
-If you wish the generated code to depend upon the very latest/bleeding-edge of galasa code, then add the `--development` flag. This will add extra settings to the `settings.gradle` file.
+If you wish the generated code to depend upon the very latest/bleeding-edge of galasa code, then add the `--development` flag. This will add extra settings to the `settings.gradle` file of the parent and the `build.gradle` file of the child test project.
```

Comment:
Maybe 'file of any child test projects.' As the tool could create >1 of them, one for each 'feature'

Reply:
Changed in latest commit